<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelectionWidgetClass</class>
 <widget class="QWidget" name="ColorSelectionWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>196</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel[text=&quot;Color Options&quot;] {
	font: bold;
}</string>
     </property>
     <property name="text">
      <string>Color Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoColorScaleCheckBox">
     <property name="text">
      <string>Automatic Color Scaling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useLogScaleCheckBox">
     <property name="text">
      <string>Use Log Scaling</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="minValLabel">
       <property name="text">
        <string>Minimum</string>
       </property>
       <property name="buddy">
        <cstring>minValLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="minValLineEdit">
       <property name="maxLength">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxValLabel">
       <property name="text">
        <string>Maximum</string>
       </property>
       <property name="buddy">
        <cstring>maxValLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maxValLineEdit">
       <property name="maxLength">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="presetButton">
       <property name="text">
        <string>Choose Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>autoColorScaleCheckBox</tabstop>
  <tabstop>minValLineEdit</tabstop>
  <tabstop>maxValLineEdit</tabstop>
  <tabstop>presetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
